packages/devtools_app/lib/src/shared/connected_app.dart

```diff
@@ -250,12 +250,6 @@ class AppState extends DisposableController with AutoDisposeControllerMixin {
   final _variables = ValueNotifier<List<DartObjectNode>>([]);
   void setVariables(List<DartObjectNode> value) => _variables.value = value;
 
-  ValueListenable<bool> get isPaused => _isPaused;
-  final _isPaused = ValueNotifier<bool>(false);
-
-  /// This setter should be invoked only by debugger.
-  void setPausedOnBreakpoint(bool value) => _isPaused.value = value;
-
   ValueListenable<Frame?> get currentFrame => _currentFrame;
```

Comment:
Is it ok for us to be removing the setPausedOnBreakpoint function in a refactor?
It sounds important

Reply:
the value here _isPaused not used any more, so setter does not make sense